lib/fuzzy-finder-view.coffee

```diff
@@ -24,6 +24,8 @@ class FuzzyFinderView extends SelectListView
         @splitOpenPath (pane, item) -> pane.splitDown(items: [item])
       'pane:split-up': =>
         @splitOpenPath (pane, item) -> pane.splitUp(items: [item])
+      'fuzzy-finder:alternate-confirm': =>
```

Comment:
This name is a bit vague - I wasn't quite sure what it did just from looking at the name. Maybe something more like `invert-confirm`? (That's probably not a great name either, but I think it should change.)